patches/chrome-browser-upgrade_detector-upgrade_detector_impl.cc.patch

```diff
@@ -10,12 +10,12 @@ index a12c9d9958376c92e8c5a78d8ab8182d5ecafd77..f29ec211b7ae2bd2bbd520cb69ad662d
      } else {
        switch_name = switches::kSimulateOutdated;
      }
-@@ -477,7 +476,7 @@ void UpgradeDetectorImpl::Init() {
+@@ -476,7 +475,7 @@ void UpgradeDetectorImpl::Init() {
```

Comment:
switch_name removal seems weird https://github.com/brave/brave-core/commit/1af9646975f07322351a14c75113d5e62ed64919#diff-e1717757fbaa5782d6822820e62e9739aa87b5c766b0560ec00e450766322be3R9

Reply:
Yes, this looks to be an accident. Removed this patch in a2aca8d72c95178c93edb8173f5940279a8b3f1d